### ISSUE TITLE
.Net core SDK not found

### ISSUE THREAD

**Issue by mattgen88** [dotnet/core#7661]
### Description

After upgrading to the latest dotnet-sdk-6.0, no sdks are found

```Package: dotnet-sdk-6.0
Version: 6.0.302-1
Priority: standard
Section: devel
Maintainer: Microsoft <dotnetcore@microsoft.com>
Installed-Size: 339 MB
Depends: dotnet-targeting-pack-6.0 (>= 6.0.7), netstandard-targeting-pack-2.1 (>= 2.1.0), aspnetcore-runtime-6.0 (>= 6.0.7), dotnet-apphost-pack-6.0 (>= 6.0.7), dotnet-runtime-6.0 (>= 6.0.7), aspnetcore-targeting-pack-6.0 (>= 6.0.7)
Homepage: https://github.com/dotnet/core
Download-Size: 84.9 MB
APT-Manual-Installed: yes
APT-Sources: https://packages.microsoft.com/ubuntu/22.04/prod jammy/main amd64 Packages
Description: Microsoft .NET SDK 6.0.302
 .NET is a development platform that you can use to build command-line applications, microservices and modern websites. It is open source, cross-platform and is supported by Microsoft. We hope you enjoy using it! If you do, please consider joining the active community of developers that are contributing to the project on GitHub (https://github.com/dotnet/core). We happily accept issues and PRs.
```

packages were removed and re-isntalled to the same result.

```
$dotnet --info

global.json file:
  Not found

Host:
  Version:      6.0.7
  Architecture: x64
  Commit:       0ec02c8c96

.NET SDKs installed:
  No SDKs were found.

.NET runtimes installed:
  Microsoft.AspNetCore.App 6.0.7 [/usr/lib/dotnet/dotnet6-6.0.107/shared/Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 6.0.7 [/usr/lib/dotnet/dotnet6-6.0.107/shared/Microsoft.NETCore.App]

Download .NET:
  https://aka.ms/dotnet-download

Learn about .NET Runtimes and SDKs:
  https://aka.ms/dotnet/runtimes-sdk-info
```

Installed packages:
```
$apt search dotnet | ack installed

WARNING: apt does not have a stable CLI interface. Use with caution in scripts.

aspnetcore-runtime-6.0/jammy-proposed,now 6.0.107-0ubuntu2~22.04.1 amd64 [installed,automatic]
dotnet-apphost-pack-6.0/jammy-proposed,now 6.0.107-0ubuntu2~22.04.1 amd64 [installed,automatic]
dotnet-host/jammy-proposed,now 6.0.107-0ubuntu2~22.04.1 amd64 [installed,automatic]
dotnet-hostfxr-6.0/jammy-proposed,now 6.0.107-0ubuntu2~22.04.1 amd64 [installed,automatic]
dotnet-runtime-6.0/jammy-proposed,now 6.0.107-0ubuntu2~22.04.1 amd64 [installed,automatic]
dotnet-sdk-6.0/jammy,now 6.0.302-1 amd64 [installed]
dotnet-targeting-pack-6.0/jammy-proposed,now 6.0.107-0ubuntu2~22.04.1 amd64 [installed,automatic]
libgtk2.0-cil/jammy,now 2.12.40-3ubuntu1 amd64 [installed,automatic]
```

### Regression?

Before upgrading to 6.0.302-1, the 6.0 sdk was available. After upgrade, I have no sdks and can no longer build applications.

**Comment by mattgen88** [comment]
Current work around is to download the binary tar.gz for SDK 6.0.302 instead of using the package manager version https://dotnet.microsoft.com/en-us/download/dotnet/6.0

Installing via package manager results in no sdks. Installing the binary, the 6.0 sdk is available.

**Comment by buyaa-n** [comment]
CC @wli3 @dsplaisted

**Comment by dsplaisted** [comment]
@MichaelSimons Any idea why this would happen?  I tried updating to the latest packages on Ubuntu 18 and it didn't give me any issues.  The package version numbers look different though:

```
dsplaisted@DEEPMINI:~$ apt search dotnet | ack installed

WARNING: apt does not have a stable CLI interface. Use with caution in scripts.

aspnetcore-runtime-6.0/jammy,now 6.0.7-1 amd64 [installed,automatic]
aspnetcore-targeting-pack-6.0/jammy,now 6.0.7-1 amd64 [installed,automatic]
dotnet-apphost-pack-6.0/jammy,now 6.0.7-1 amd64 [installed,automatic]
dotnet-host/jammy,now 6.0.7-1 amd64 [installed,automatic]
dotnet-hostfxr-6.0/jammy,now 6.0.7-1 amd64 [installed,automatic]
dotnet-runtime-6.0/jammy,now 6.0.7-1 amd64 [installed,automatic]
dotnet-runtime-deps-6.0/jammy,now 6.0.7-1 amd64 [installed,automatic]
dotnet-sdk-6.0/jammy,now 6.0.302-1 amd64 [installed]
dotnet-targeting-pack-6.0/jammy,now 6.0.7-1 amd64 [installed,automatic]
netstandard-targeting-pack-2.1/jammy,now 2.1.0-1 amd64 [installed,automatic]
```

**Comment by MichaelSimons** [comment]
Please see https://github.com/dotnet/core/issues/7699 for guidance on how to resolve this situation.

**Comment by mattgen88** [comment]
Resolved by #7699 confirmed

**Comment by mairaw** [comment]
So I'll close the bug. Thanks for letting us know.
